eval/eval_chunking_grading.py

```
@@ -352,23 +910,45 @@ def has_key(m):
                     err = abs(ai_score - row["prof_score"])
                     errors.append(err)
-                    print(f"    Row {i+1}: prof={row['prof_score']} ai={ai_score:.1f} err={err:.1f}")
+                    print(f"    Row {i+1}: prof={row['prof_score']} ai={ai_score:.1f} err={err:.1f}", flush=True)
             if errors:
                 mae = sum(errors) / len(errors)
                 key = f"{strategy}+{model_name}"
                 results[key] = {"mae": mae, "n": len(errors)}
-                print(f"    MAE = {mae:.2f} (n={len(errors)})")
+                print(f"    MAE = {mae:.2f} (n={len(errors)})", flush=True)
+            else:
+                print(f"    No successful grades. Check API key and model id.", flush=True)
```

Ruff (0.15.9)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)